reports/report_builder.py

```diff
@@ -432,6 +433,9 @@ def apply_weights(self, rows, db_table, media_type):
             weights = 'SELECT cast(SUM(reports_weights.weight) as float) AS "n",'
 
         raw_query = raw_query.replace('SELECT', weights)
+        if raw_query.find('GROUP BY') == -1:
```

Comment:
Some `raw_query` was coming in without the `'GROUP BY'`

Reply:
I'd say lets try to figure out why `GROUP BY` isn't included in some `raw_query` first @esirK .

Reply:
@kilemensi  The reason why some `raw_query` were missing `GROUP BY` is because the `apply_weights` functionality was added later when some sheets had already been generated. From the code, we can see that all new sheets that were being added included a `.annotate(n=Count('id'))` method which is responsible for the `GROUP BY`.

Reply:
Cool col @esirK ... but lets speak English. When you say:

> `apply_weights` was added later when some sheets had already been generated

What does this exactly mean?